chapter_convolutional-modern/batch-norm.md

```diff
@@ -208,10 +208,12 @@ class BatchNorm(nn.Block):
 
     def forward(self, X):
         # 如果 `X` 不在内存上，将 `moving_mean` 和 `moving_var`
-        # 复制到 `X` 所在显存上
+        # 以及参数 `gamma` 和 `beta` 复制到 `X` 所在显存上
```

Comment:
Since `gamma` and `beta` won't be updated (as moving mean or var), can you elaborate why you want to copy to device every time?

Reply:
According to my understanding, I want to execute my code like this, but an exception will occur.
```python
bn = BatchNorm(num_featurs, num_dims)
x = torch.randn(shape, device='cuda')
bn(x)
```
so the correct way is as follows, right?

```python
bn = BatchNorm(num_featurs, num_dims)
bn = bn.to(device='cuda') # copy  parameter to gpu
x = torch.randn(shape, device='cuda')
bn(x)
```